### PR TITLE
Feature/settings pages

### PR DESCRIPTION
**Pull request by Yarrow228**
Setings navigation menu.
Settings pages: Privacy Policy, About us, Terms of use, Contact support(with form validation).

### CODE REVIEW

File: src/app/app-routing.ts [hunk 1]
```diff
@@ -49,6 +54,28 @@ const routes: Routes = [
     path: 'profile',
     component: ProfileComponent,
   },
+  {
+    path: 'settings',
```
Comment (Semenone) [inline src/app/app-routing.ts:58]: At all paths, you need to use enums for names, Example: AppPaths.SETTINGS. What is this for i hope you say me...

File: src/app/components/contact-support/contact-support.component.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  </div>
+  <div class="contact-support__contact-info">
+    <span class="contact-support__phone-icon"></span
+    ><span class="contact-support__contact-text">(405) 555-0128</span>
+  </div>
+  <div class="contact-support__contact-info">
+    <span class="contact-support__email-icon"></span
+    ><span class="contact-support__contact-text"
+      >jessica.hanson@example.com</span
+    >
```
Comment (Semenone) [inline src/app/components/contact-support/contact-support.component.html:61]: ?

File: src/app/components/contact-support/contact-support.component.scss [hunk 3]
```diff
@@ -0,0 +1,109 @@
+@import '../../../styles/vars';
```
Comment (Semenone) [inline src/app/components/contact-support/contact-support.component.scss:1]: are you sure you can't set the path any other way?

File: src/app/components/contact-support/contact-support.component.scss [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    width: 29px;
+    height: 30px;
+    background: url('~assets/images/phone-icon.svg');
+  }
+
+  &__email-icon {
+    display: block;
+    width: 32px;
+    height: 22px;
+    background: url('~assets/images/email-icon.svg');
```
Comment (Semenone) [inline src/app/components/contact-support/contact-support.component.scss:91]: You not have sass constants for images path?

File: src/app/components/contact-support/contact-support.component.ts [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'contact-support',
+  templateUrl: './contact-support.component.html',
+  styleUrls: ['./contact-support.component.scss'],
+})
+export class ContactSupportComponent implements OnInit {
+  form: FormGroup;
```
Comment (Semenone) [inline src/app/components/contact-support/contact-support.component.ts:10]: where accessors?

File: src/app/components/contact-support/contact-support.component.ts [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+      email: new FormControl('', [Validators.email, Validators.required]),
+      message: new FormControl('', [
+        Validators.required,
+        Validators.minLength(2),
+        Validators.maxLength(255),
+      ]),
+    });
+  }
+
+  submit() {
```
Comment (Semenone) [inline src/app/components/contact-support/contact-support.component.ts:30]: see above

File: src/app/components/footer/footer.component.html [hunk 7]
```diff
@@ -9,14 +9,14 @@ <h4 class="footer__list-header">Browse</h4>
         </ul>
         <ul>
           <li><a href="#">Help</a></li>
-          <li><a href="#">Contact Support</a></li>
+          <li><a routerLink="/settings/contact-support">Contact Support</a></li>
```
Comment (Semenone) [inline src/app/components/footer/footer.component.html:12]: Need use constant

File: src/app/components/profile-dropdown-menu/profile-dropdown-menu.component.html [hunk 8]
```diff
@@ -17,7 +17,7 @@
       <span class="menu__profile-icon"></span>Profile
     </button>
     <div class="menu__second-line"></div>
-    <button class="menu__option-btn">
+    <button routerLink="/settings/about-us" class="menu__option-btn">
```
Comment (Semenone) [inline src/app/components/profile-dropdown-menu/profile-dropdown-menu.component.html:20]: see above

File: src/assets/images/email-icon.svg [hunk 9]
```diff
@@ -0,0 +1,3 @@
+<svg width="32" height="22" viewBox="0 0 32 22" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Semenone) [inline src/assets/images/email-icon.svg:1]: Why you need to use svg?

Comment (Yarrow228) [reply src/assets/images/email-icon.svg:1]: Why exactly .svg?

File: src/app/components/contact-support/contact-support.component.html (hunk 2 above)
Comment (Yarrow228) [reply src/app/components/contact-support/contact-support.component.html:61]: ?

File: src/app/pages/settings/settings.component.scss [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+  display: flex;
+  flex-direction: column;
+
+  &__link {
+    font-size: 16px;
+    line-height: 19px;
+    white-space: nowrap;
+
+    padding-left: 4px;
+    padding-right: 4px;
```
Comment (marinbonibon) [inline src/app/pages/settings/settings.component.scss:29]: It can be shorter

File: src/app/components/footer/footer.component.html (hunk 7 above)
Comment (Yarrow228) [reply src/app/components/footer/footer.component.html:12]: > Need use constant

?

File: src/app/components/contact-support/contact-support.component.html (hunk 2 above)
Comment (Semenone) [reply src/app/components/contact-support/contact-support.component.html]: why do you move such brackets?

Comment (Yarrow228) [reply src/app/components/contact-support/contact-support.component.html]: > why do you move such brackets?

It's  done by 'prettier'

**Pull request merged**